dlme_airflow/models/collection.py

```diff
@@ -1,4 +1,4 @@
-from utils.catalog import catalog_for_provider
+from dlme_airflow.utils.catalog import catalog_for_provider
```

Comment:
@edsu is there any reason not to include `dlme_airflow` in this include, and if not how to avoid it?

It works fine in the app as-is, but the test throws an error without the `dlme_airflow`.

Reply:
As you saw over in #212 I think it might be easier if we used `dlme_airflow` in our imports, that way we won't need to remember to set PYTHONPATH to `dlme_airflow`?  I think the app is working fine because it burns PYTHONPATH into the container environment.